### PR TITLE
Add Signup Page

### PR DESCRIPTION
**Pull request by arnoldbgm**
# Description

Implementation of the signup page, installation of the Roboto font and new svg images.

Fixes # (issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

image

### CODE REVIEW

File: src/assets/index.ts [hunk 1]
```diff
@@ -1 +1,4 @@
 export { default as logo } from "./logo/X_black.svg";
+export { default as logoGray } from "./logo/X_gray.svg";
+export { default as apple } from "./logo/apple.svg";
```
Comment (linder3hs) [inline src/assets/index.ts]: los logos de apple y google debería ir en otra carpeta como `social-media`

File: src/pages/signup/index.tsx [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    'Developers',
+    'Directory',
+    'Settings',
+    '© 2021 Twitter Inc.',
+  ];
+
+  return (
+    <div className='flex flex-col h-screen'>
+      <div className='flex-grow flex'>
+        {/* Black logo starts */}
```
Comment (linder3hs) [inline src/pages/signup/index.tsx]: eliminar comentarios

File: src/pages/signup/index.tsx [hunk 3]
```diff
@@ -0,0 +1,96 @@
+import { apple, google, logo, logoGray } from '../../assets';
+
+export default function SignupPage() {
+  const footerLinks = [
```
Comment (linder3hs) [inline src/pages/signup/index.tsx]: el footer debe ser parte del layout

File: src/pages/signup/index.tsx [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+                Sign up with Apple
+              </button>
+              <button className='w-[403px] h-[62px] bg-transparent border border-gray-300 border-opacity-80 rounded-full font-roboto text-base font-normal leading-normal flex items-center justify-center hover:bg-gray-200 hover:opacity-100 transition duration-300'>
+                Sign up with phone or email
+              </button>
+            </div>
+            <div className='w-[373px] mt-[15px]'>
+              <p className='font-roboto'>
+                By singing up you agree to the
+                <span className='text-[#1E97E1] cursor-pointer'> Terms of Service </span>
```
Comment (linder3hs) [inline src/pages/signup/index.tsx]: los colores como `#1E97E1` deben ser agregados al archivo de tailwind

File: src/pages/signup/index.tsx [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+                <img
+                  src={apple}
+                  alt='Apple'
+                  width={31}
+                  height={32}
+                  className='mr-2'
+                />
+                Sign up with Apple
+              </button>
+              <button className='w-[403px] h-[62px] bg-transparent border border-gray-300 border-opacity-80 rounded-full font-roboto text-base font-normal leading-normal flex items-center justify-center hover:bg-gray-200 hover:opacity-100 transition duration-300'>
```
Comment (linder3hs) [inline src/pages/signup/index.tsx]: usar medidas fijas en px no es recomendable recuerda que tailwind es pensado en mobile first

**Review by linder3hs — CHANGES_REQUESTED** [review]

File: src/assets/index.ts (hunk 1 above)
Comment (arnoldbgm) [reply src/assets/index.ts]: Solucionado 😎

File: src/pages/signup/index.tsx (hunk 2 above)
Comment (arnoldbgm) [reply src/pages/signup/index.tsx]: Solucionado 😎😎

File: src/pages/signup/index.tsx (hunk 3 above)
Comment (arnoldbgm) [reply src/pages/signup/index.tsx]: No me sale, me da un scroll inncesario, ayuda ahi sensei :(

File: src/pages/signup/index.tsx (hunk 4 above)
Comment (arnoldbgm) [reply src/pages/signup/index.tsx]: Solucionado 😎

File: src/pages/signup/index.tsx (hunk 5 above)
Comment (arnoldbgm) [reply src/pages/signup/index.tsx]: Solucionado 😎

**Pull request merged**